grc/core/params/dtypes.py

```diff
@@ -70,10 +70,12 @@ def validate_block_id(param):
 
 @validates('name')
 def validate_name(param):
-    # Name of a function that will be generated literally not as a string
+    # Name of a function or other block that will be generated literally not as a string
     value = param.value
+
+    # Allow blank to pass validation
     # Can python use this as a variable?
-    if not re.match(r'^[a-z|A-Z]\w*$', value):
+    if not re.match(r'^([a-z|A-Z]\w*)?$', value):
```

Comment:
Since you're touching the regex, it's broken: Since we're in Python3, it will match way too broadly. This needs the ASCII flag.

Reply:
will redo this with the additonal attribute as mentioned above as a separate pr rather than opening up the regex for this dtype.  closing this one for now

the other option is just to set the dtype to raw and lose parameter validation which isn't really thorough in this case anyway.